### PR TITLE
Object handling around Temporal.Calendar.from().

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: polyfill/lib/ecmascript.mjs [hunk 1]
```diff
@@ -576,7 +576,11 @@ export const ES = ObjectAssign({}, ES2019, {
     if (from === undefined) {
       from = GetIntrinsic('%Temporal.Calendar.from%');
     }
-    return ES.Call(from, TemporalCalendar, [calendarLike]);
+    const calendar = ES.Call(from, TemporalCalendar, [calendarLike]);
+    if (ES.Type(calendar) !== 'Object') {
+      throw new TypeError('Temporal.from should return an object');
```
Comment (ryzokuken) [inline polyfill/lib/ecmascript.mjs:581]:
```suggestion
      throw new TypeError('Temporal.Calendar.from should return an object');
```

**Review by ryzokuken — APPROVED** [review]
Just one comment, thanks.

File: polyfill/lib/ecmascript.mjs [hunk 2]
```diff
@@ -57,7 +58,8 @@ const ES2019 = {
   ToNumber,
   ToObject,
   ToPrimitive,
-  ToString
+  ToString,
+  Type
```
Comment (ptomato) [inline polyfill/lib/ecmascript.mjs:62]: This is also in https://github.com/tc39/proposal-temporal/pull/855, if you would like to review that

**Review by ptomato — APPROVED** [review]
:+1:

**Pull request merged**